org.gnome.OfficeRunner.json

```diff
@@ -0,0 +1,34 @@
+{
+    "app-id": "org.gnome.OfficeRunner",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "office-runner",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* systemd */
+        "--system-talk-name=org.freedesktop.login1",
+        /* upower */
+        "--system-talk-name=org.freedesktop.UPower",
+        /* Play sounds */
+        "--socket=pulseaudio"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g"
+    },
+    "modules": [
+        {
+            "name": "office-runner",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://git.gnome.org/office-runner"
```

Comment:
this needs to be a reproducible source. so a git tag+commit or a release archive to be available on flathub.

Reply:
That's done. It would have been replaced by a release once it got merged.